spec/jobs/deposit_job_spec.rb

```diff
@@ -14,43 +14,45 @@
   end
 
   context 'when a new work' do
-    let(:work_form) { WorkForm.new(title: work.title, content_id: content.id) }
-    let(:work) { create(:work, :deposit_job_started) }
+    let(:form) { WorkForm.new(title: object.title, content_id: content.id) }
```

Comment:
This needs specs for collections.

Reply:
Yeah, I was doing a collection deposit in my follow up PR but being one job, I'll pull that spec in.